index.html

```diff
@@ -94,7 +143,7 @@ <h3 class="sub-heading">Multi-Post Stories</h3>
               <li class="lang-tag">bootstrap</li>
```

Comment:
- [ ] Kindly remove your project cards from your html and generate them dynamically from your javascript file to follow the requirements :blush:

Reply:
did that and I explained that I commented it out